Description: Fix path for the /etc/target/saveconfig.json
 The default path is /etc/target, which IMO is a way too generic. So I changed
 this to /etc/rtslib-fb-target/saveconfig.json
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-01-25

Index: python-rtslib-fb/scripts/targetctl
===================================================================
--- python-rtslib-fb.orig/scripts/targetctl
+++ python-rtslib-fb/scripts/targetctl
@@ -28,7 +28,7 @@ from rtslib_fb import RTSRoot
 import os
 import sys
 
-default_save_file = "/etc/target/saveconfig.json"
+default_save_file = "/etc/rtslib-fb-target/saveconfig.json"
 err = sys.stderr
 
 def usage():
Index: python-rtslib-fb/rtslib/root.py
===================================================================
--- python-rtslib-fb.orig/rtslib/root.py
+++ python-rtslib-fb/rtslib/root.py
@@ -29,7 +29,7 @@ from .tcm import so_mapping, StorageObje
 from .utils import RTSLibError, modprobe, mount_configfs
 from .utils import dict_remove, set_attributes
 
-default_save_file = "/etc/target/saveconfig.json"
+default_save_file = "/etc/rtslib-fb-target/saveconfig.json"
 
 class RTSRoot(CFSNode):
     '''
